[RUN-5271] Rename tracing-in-runtime to opentelemetry-in-runtime (#11499)
mendix/docs: content/en/docs/deployment/mendix-cloud-deploy/monitoring-with-apm/datadog-metrics.md

## Mendix Tracing{#mendix-tracing}

> Mendix Tracing uses [OpenTelemetry](/refguide/opentelemetry-in-runtime/) to send Mendix Runtime traces to [Datadog APM](https://docs.datadoghq.com/tracing/), giving you distributed tracing visibility into your app's performance and microflow executions.

Reviewer: Perhaps...

```suggestion
Mendix uses [OpenTelemetry](/refguide/opentelemetry-in-runtime/) to send Mendix Runtime traces to [Datadog APM](https://docs.datadoghq.com/tracing/), giving you a preview of your application performance and microflow executions.
```

Author: "giving you insight into" perhaps? I don't like the word "preview" because it might imply that there's a more elaborate view of your apps performance, which there is not.
@MarkvanMents what do you think?

Reviewer: Sure. That will also do.

Reviewer 2: Does this look good?

```suggestion
Mendix uses [OpenTelemetry](/refguide/opentelemetry-in-runtime/) to send Mendix Runtime traces to [Datadog APM](https://docs.datadoghq.com/tracing/), giving you an insight into your app's microflow executions and performance.
```
